turbopack/crates/turbopack-core/src/resolve/mod.rs

```diff
@@ -1084,7 +1084,7 @@ async fn imports_field(lookup_path: Vc<FileSystemPath>) -> Result<Vc<ImportsFiel
         return Ok(ImportsFieldResult::None.cell());
     };
 
-    let read = read_package_json(*package_json_path).await?;
+    let read = read_package_json(**package_json_path).await?;
```

Comment:
Why do so many places now need a `**`?

Reply:
`package_json_path` is now a `&ResolvedVc<FileSystemPath>`. The first deref is used to get a `ResolvedVc<FileSystemPath>`. The second deref is used to get a `Vc<FileSystemPath>`.

`&ResolvedVc` doesn't make sense if you can avoid it, because `ResolvedVc` is `Copy`, but it's not always easy to avoid. In this case, it's because `package_json_context` is a `ReadRef`, which only gives a reference to the underlying data.